lib/extensions/ar_adapter/ar_dba.rb

```diff
@@ -462,6 +462,7 @@ def table_size
                   FROM pg_class
                  WHERE reltuples > 1
                    AND relname NOT LIKE 'pg_%'
+                   AND relkind = 'r'
```

Comment:
I made the above two changes to eliminate system tables and only include ordinary tables and not indexes, etc.

Reply:
may want to `reltuples > 0` to only display rows with a value

Reply:
Instead of `WHERE reltuples > 1` that's there?  I guess I can.  It's basically the same thing, the idea was to keep the original query and only normal tables and not indexes.

Reply:
sorry - didn't see that. keep what is there